lab_1/main.py

```diff
@@ -11,26 +11,66 @@ def tokenize(text: str) -> list or None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
-    pass
 
+    if not (isinstance(text, str)):  # text не является экземпляром классом str
+        return None
+    else:
```

Comment:
you don't even need to write `else`
can you remove `else` here and further?

the code after `return` won't work, so we don't need it